### DISCUSSION TITLE
JLReq TF meeting notes 2024-11-12

### DISCUSSION THREAD

**Discussion by kidayasuo** [w3c/jlreq#448] [meetingnotes]
# JLReq TF meeting notes 2024-11-12
（[日本語](https://github.com/w3c/jlreq/discussions/448#discussioncomment-11390339)）

### attendees
atsushi, kida, murata, suzuki, tajima, tatsuo, toshi, yamamoto, yamashige

### Previous Meetings
[JLReq TF meeting Notes 2024-10-01](https://github.com/w3c/jlreq/discussions/447)
- Discussed the Japanese layout model on the baseline system
- New document license terms
- Reviewed some GH issues

[JLReq-d Writing meeting 2024-10-9](https://github.com/w3c/jlreq-d/discussions/66)
- Drafted "3.2 the Japanese layout model on the baseline system" based on the discussion on 10-01
- Reviewed the section 1 and 2 up to 2.2 Hiragana and Katakana

[JLReq-d Writing meeting 2024-10-24](https://github.com/w3c/jlreq-d/discussions/68)
- Restructured the section 2; separated out line-layout related section as the section 3.
- Reviewed section 2.2 and 2.3.

[JLReq-d Writing meeting 2024-10-31](https://github.com/w3c/jlreq-d/discussions/71)
- Drafted 2.3.
- Reviewed 2.4 up to "Japanese typefaces"
- All changes for the section 2 has been checked in.

### Agenda for this meeting
- JLReq TF Charter
- JLReq document maintenance release
- Status Update: ISO Open Font Format 5th Edition
- Team updates
- The next meeting

### Time to update the [JLReq TF Charter](https://www.w3.org/International/jlreq/charter/)
The end date is 31 December 2024

#### Reviewed the current Deliverables which are shown below:
1. Augment the contents of the [gap-analysis document](https://www.w3.org/TR/jpan-gap/), which is one of many similar documents that radiate out from the [language matrix](https://www.w3.org/International/typography/gap-analysis/language-matrix.html) to help us track areas where better support is needed. The gap analysis document uses a standard structure to list issues that content users and content authors face relating to particular apects of text layout, provide pointers to tests, and prioritise the importance of the barriers documented.
2. Develop the [Requirements for Japanese Digital Text Layout](https://github.com/w3c/jlreq-d/) document to capture requirements and new items to be considered in digital text, better accessibility using digital features, and necessity technologies for Japanese text layout in internalionalized environment.
3. Develop the [Text to Speech of Electronic Documents Containing Ruby: User Requirements](https://w3c.github.io/ruby-t2s-req/) to capture accessibility user requirements for Ruby.
4. Update the [Japanese Layout Requirements](https://www.w3.org/TR/jlreq/) document to fix errata.
5. Update the [Rules for Simple Placement of Japanese Ruby](https://www.w3.org/TR/simple-ruby/) document to fix errata.
6. Produce other documents advising standards developers and application developers on how to cater for Japanese needs. Some of those documents may be developed elsewhere, such as at the Advanced Publishing Lab in Japan, and brought to this group for review and subsequent publication.

#### Discussions
1. The **Gap-Analysis** section will remain as-is. Although there hasn’t been much recent activity in this area, we plan to focus on it more actively once the development of **jlreq-d** stabilizes.
2. The development of **jlreq-d** continues to be one of the group’s main activities.
3. The **Ruby T2S** is nearing its first publication. This item will remain.
4. **JLReq** is in maintenance mode. Simplify the wording to something like: "Maintain Japanese Layout Requirements document."
5. The **Simple Ruby** document has been discontinued.
6. Remove the following part as we are no longer affiliated with APL: *", such as at the Advanced Publishing Lab in Japan,"*.

**todo:** Shimono-san to create the SOTD document.
**todo:** Kida to ask xfq if the "Japanese Script Resource Page" should be mentioned.
**todo:** Review the remaining parts of the Charter document in the next meeting.

### JLReq document maintenance release
A maintenance update for JLReq is planned. Shimono-san still has some tasks to complete. This release aims to address issues such as:
- [Update lang switch for recent (3y) respec html change #419](https://github.com/w3c/jlreq/pull/419)
- [Update URL when swithing the language in UI #262](https://github.com/w3c/jlreq/pull/262)

#### Reviewed issues filed against JLReq document
These issues, mostly enhancement requests, will not be fixed:
- [Switch the language of the title #321](https://github.com/w3c/jlreq/issues/321)
- [anchors appear to have been broken in some update #251](https://github.com/w3c/jlreq/issues/251)
- [Optimize the images #218](https://github.com/w3c/jlreq/issues/218)
- [The page needs to open in English by default #126](https://github.com/w3c/jlreq/issues/126)
- [Sort the glossary by one language or another #93](https://github.com/w3c/jlreq/issues/93)

We will plan addressing them in jlreq-d
- [Description on Japanese fonts and glyph #160](https://github.com/w3c/jlreq/issues/160)
- [How to add ID to an element? #154](https://github.com/w3c/jlreq/issues/154)
- [Where does underline go relative to ruby? #9](https://github.com/w3c/jlreq/issues/9) (added comments from the meeting)

**todo:** Review all issues labeled `jlreq-doc:*`. Most will either be closed or deferred to **jlreq-d**.

### Status Update: ISO Open Font Format 5th Edition
The development of the 5th edition of the Open Font Format, ISO/IEC 14496-22, is underway and has reached the Committee Draft (CD) stage. The ballot period has ended, and the working group is now preparing an updated draft based on the comments received. The team is planning to advance the document to the next stage, Draft International Standard (DIS).
[https://www.iso.org/standard/87621.html](https://www.iso.org/standard/87621.html)

### Team updates
Suzuki-san @ FontWorks is leaving the company but will remain in JLReq TF. Needs to update his email address. He will also contact other members in FontWorks to see if they want to join JLReq TF.

**Comment by kidayasuo** [comment]
## Agenda

### 前回の会議
[JLReq TF Meeting Notes 2024-10-01](https://github.com/w3c/jlreq/discussions/447#discussioncomment-11176950)
- ベースラインシステムにおける日本語のレイアウトモデルについて議論
- 新しい文書のライセンス条件
- 一部のGitHubの課題をレビュー

[JLReq-d 編集会議 2024-10-9](https://github.com/w3c/jlreq-d/discussions/66)
- 2024-10-01の議論に基づいて「3.2 ベースラインシステムにおける日本語のレイアウトモデル」をドラフト
- セクション1と2（2.2 平仮名と片仮名まで）をレビュー

[JLReq-d 編集会議 2024-10-24](https://github.com/w3c/jlreq-d/discussions/68)
- セクション2を再構成し、行レイアウト関連のセクションをセクション3として分離
- セクション2.2と2.3をレビュー

[JLReq-d 編集会議 2024-10-31](https://github.com/w3c/jlreq-d/discussions/71)
- セクション2.3をドラフト
- 「日本語書体」までのセクション2.4をレビュー
- セクション2のすべての変更を反映済み

### 今回の議題案
- JLReq TF チャーター
- JLReq 文書メンテナンスリリース
- 状況報告：ISO Open Font Format 第5版
- チームの最新情報
- 次回の会議

## Meeting Notes

### [JLReq TF チャーター](https://www.w3.org/International/jlreq/charter/) の更新時期
終了日は2024年12月31日

#### 現在の"Deliverables"の内容
1. Augment the contents of the [gap-analysis document](https://www.w3.org/TR/jpan-gap/), which is one of many similar documents that radiate out from the [language matrix](https://www.w3.org/International/typography/gap-analysis/language-matrix.html) to help us track areas where better support is needed. The gap analysis document uses a standard structure to list issues that content users and content authors face relating to particular apects of text layout, provide pointers to tests, and prioritise the importance of the barriers documented.
2. Develop the [Requirements for Japanese Digital Text Layout](https://github.com/w3c/jlreq-d/) document to capture requirements and new items to be considered in digital text, better accessibility using digital features, and necessity technologies for Japanese text layout in internalionalized environment.
3. Develop the [Text to Speech of Electronic Documents Containing Ruby: User Requirements](https://w3c.github.io/ruby-t2s-req/) to capture accessibility user requirements for Ruby.
4. Update the [Japanese Layout Requirements](https://www.w3.org/TR/jlreq/) document to fix errata.
5. Update the [Rules for Simple Placement of Japanese Ruby](https://www.w3.org/TR/simple-ruby/) document to fix errata.
6. Produce other documents advising standards developers and application developers on how to cater for Japanese needs. Some of those documents may be developed elsewhere, such as at the Advanced Publishing Lab in Japan, and brought to this group for review and subsequent publication.

#### 議論
1. **ギャップ分析**セクションは現状のままとする。最近の活動は少ないが、**jlreq-d**の開発が安定したら、時間を割けるようになるだろう。
2. **jlreq-d**の開発はグループの主要な活動の1つとして継続。
3. **Ruby T2S** は最初の公開に近づいており、引き続き進める。
4. **JLReq** はメンテナンスモード中。簡潔な文言に変更：「日本語組版要件文書を維持」。
5. **Simple Ruby** 文書は廃止。
6. APLに関する記述（例：「日本の先進出版研究所で開発された」）を削除。

**todo:** 下野さんがSimple Ruby のSOTD文書を作成する。
**todo:** 木田さんがxfqさんに「Japanese Script Resource Page」について言及すべきか確認。
**todo:** 次回の会議でチャータードキュメントの残り部分をレビュー。

### JLReq 文書メンテナンスリリース
JLReqのメンテナンスアップデートが計画されている。下野さんに残作業がいくつかある。このリリースでは以下の問題を解決する予定：
- [最近のReSpec HTML変更に対応する言語切り替えの更新 #419](https://github.com/w3c/jlreq/pull/419)
- [UIで言語切り替え時のURLを更新 #262](https://github.com/w3c/jlreq/pull/262)

#### JLReq 文書に提起された問題のレビュー
主に拡張リクエストとしての以下の問題は修正されない：
- [タイトルの言語を切り替える #321](https://github.com/w3c/jlreq/issues/321)
- [更新で一部のアンカーが壊れた #251](https://github.com/w3c/jlreq/issues/251)
- [画像を最適化 #218](https://github.com/w3c/jlreq/issues/218)
- [デフォルトでページを英語で開く必要がある #126](https://github.com/w3c/jlreq/issues/126)
- [用語集を特定の言語でソート #93](https://github.com/w3c/jlreq/issues/93)

これらは**jlreq-d**で対応を計画：
- [日本語のフォントと字形の説明 #160](https://github.com/w3c/jlreq/issues/160)
- [要素にIDを追加する方法 #154](https://github.com/w3c/jlreq/issues/154)
- [ルビの下線の位置 #9](https://github.com/w3c/jlreq/issues/9)（会議でのコメントを追加した）

**todo:** `jlreq-doc:*` ラベル付きの全ての問題をレビュー。大半はクローズまたは**jlreq-d**に移行予定。

### 状況報告：ISO Open Font Format 第5版
ISO/IEC 14496-22、Open Font Format 第5版の開発が進行中で、現在は委員会原案（CD）段階にある。投票期間が終了し、作業グループは寄せられたコメントに基づいて更新された原案を準備中。この文書は次の段階である国際標準原案（DIS）に進む予定。
[https://www.iso.org/standard/87621.html](https://www.iso.org/standard/87621.html)

### チームの最新情報
FontWorksの鈴木さんが退社される。JLReq TFには引き続き参加予定。メールアドレスを更新する必要がある。また、FontWorksの他のメンバーにJLReq TF参加の意向を確認する予定。

### 次回の会議は2024-12-3

**Comment by yamahige** [comment]
> - [ルビの下線の位置 #9](https://github.com/w3c/jlreq/issues/9)（会議でのコメントを追加した）

> Yamaguchi-san suggested that authors could resolve the issue by specifying the ordering using CSS or by defining the distances between the main text and the annotations.
> [ルビの下線の位置 #9](https://github.com/w3c/jlreq/issues/9#issuecomment-2499348163)

やってみました。ルビと下線（傍線？）（加えて圏点）の位置、オフセット、背景色などをCSSで変えて試せるページを作ってみました:
[ルビと下線と圏点の調整](https://yamahige.github.io/study/ruby_underline_emphasis/ruby_underline.html)
ブラウザーによって挙動が異なります。

次のような議論の手がかりにいかがでしょう
- 調整しても満足できない
- 調整によって満足できるレイアウトになるからよし
- 調整によって満足できるレイアウトになるが、既定値は◯◯であってほしい
